test/Libraries/RevitIntegrationTests/AdaptiveComponentTests.cs

```diff
@@ -181,6 +182,10 @@ public void CreateAdaptiveComponentsByPointsOnFace()
                 var adapValue = GetPreviewValueAtIndex(adapID, i) as AdaptiveComponent;
                 Assert.IsNotNull(adapValue);
             }
+            var volumeSumId = "f76b55af-52f0-451e-9819-2a51ce8a1afc";
+            var volume = GetPreviewValue(volumeSumId);
+            Assert.IsTrue(Math.Abs((double)volume - 13.39) < 0.1);
```

Comment:
Hey @mjkkirschner, can you elaborate a little on this assertion?  Also does `AdaptiveComponentsByPointsOnFace.dyn` get used for any other tests that may be influenced by the changes made to the graph?